.claude/commands/feature.md

```
@@ -0,0 +1,254 @@
+# /feature - Feature/UI Layer Implementation
+
+## Purpose
+Implement the feature/UI layer including ViewModel, Screen, Components, and Navigation.
+
+---
+
+## Workflow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)